gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/proc/LaunchDagProc.java

```diff
@@ -84,8 +84,6 @@ protected void act(DagManagementStateStore dagManagementStateStore, Optional<Dag
       // todo - add metrics
     } else {
       DagProcUtils.submitNextNodes(dagManagementStateStore, dag.get(), getDagId());
-      // Checkpoint the dag state, it should have an updated value of dag nodes
```

Comment:
why do we not update the store here? is it done in conclude or only needs to be done once in initialize?

Reply:
because DagProcUtils.submitNextNodes updates the table already